test_helper.go

```diff
@@ -97,6 +97,36 @@ func RecordConformanceTest(t *testing.T, create func() Record) {
 		}
 	})
 
+	t.Run("copy just parsed + Get", func(t *testing.T) {
+		values := map[FieldIndex]string{
+			0:   "foo",
+			1:   "bar",
+			260: "baz",
+		}
+		org := create()
+		for k, v := range values {
+			org.Set(k, []byte(v))
+		}
```

Comment:
Not sure this is more easy/readable to create a map just to loop on it, rather than calling 3 times `org.Set()`
and 3 time `bytes.Equal` at the end of the test.
Also for consistency with other subtests